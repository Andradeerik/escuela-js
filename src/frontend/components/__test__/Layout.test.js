import React from 'react';
import { create } from 'react-test-renderer';
import Layout from '../Layout';

describe('Layout component', () => {
  test('Matches the snapshot', () => {
    const layout = create(<Layout />);
    expect(layout.toJSON()).toMatchSnapshot();
  });
});
