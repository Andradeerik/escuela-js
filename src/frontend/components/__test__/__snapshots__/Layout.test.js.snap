// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component Matches the snapshot 1`] = `
<div
  className="App"
>
  <footer
    className="footer"
  >
    <a
      href="/"
    >
      Terminos de uso
    </a>
    <a
      href="/"
    >
      Declaraci√≥n de privacidad
    </a>
    <a
      href="/"
    >
      Centro de ayuda
    </a>
  </footer>
</div>
`;
